forte/data/readers/openie_reader.py

```diff
@@ -42,11 +42,23 @@ class OpenIEReader(PackReader):
         The related source code for generating this dataset can be found
         `here
         <https://github.com/gabrielStanovsky/oie-benchmark>`__.
-        To use this Reader, you must follow the dataset format described
-        `here
+        To use this Reader, you must follow the dataset format. Each line in
+        the dataset should contain following fields:
+
+        .. code-block:: none
+
+            <sentence>\t<predicate_head>\t<full_predicate>\t<arg1>\t<arg2>....
+
+        You can also find the dataset format `here
         <https://github.com/gabrielStanovsky/oie-benchmark/tree/master/oie_corpus>`__.
     """
 
+    def initialize(self, resources: Resources, configs: Config):
+        super().initialize(resources, configs)
+
+        if configs.oie_file_extension is None:
+            raise ProcessorConfigError("OIE dataset file extension not found")
```

Comment:
```suggestion
            raise ProcessorConfigError("Configuration oie_file_extension not provided.")
```

Currently this sounds a bit like `FileNotFound`.

Reply:
Done. Thanks. This is more meaningful.